### PR TITLE
create login-page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
add two section
1. for image
2. login-form
Screenshot (91)

### CODE REVIEW

File: login-page/login-page.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    outline: none;
+  }
+
+.login-form-tag{
+    font-size: 2rem;
+    margin: auto;
+}
+.sign-up-link{
+    font-size: 1.5rem;
+    color: red;
```
Comment (yashpatel1797) [inline login-page/login-page.css:54]: use variable here.

File: login-page/login-page.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline login-page/login-page.html:6]: add author meta tag.

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge

File: login-page/login-page.html (hunk 2 above)
Comment (agrawalshanu708) [reply login-page/login-page.html:6]: ohkay yash

Comment (therajatg) [reply login-page/login-page.html:6]: add the description meta tag

**Review by therajatg — APPROVED** [review]
add description meta tag everything else looks good

**Pull request merged**